### ISSUE TITLE
performance image modules: inception

### ISSUE THREAD

**Issue by jdvylder** [tensorflow/hub#96]
Hi all, 
this is more of a question than an issue, but since it's really about the implementation of the module, this seems to be the right place to ask. 

I'm currently using a module that generates a feature vector based on inception v3 (https://www.tensorflow.org/hub/modules/google/imagenet/inception_v3/feature_vector/1 ) which is based on the tf-slim implementation. 
When I run this I get a about 9 global steps/second which is much better than when I just run the tf-slim network (that one runs at 3 steps/second). I do like this gain in performance, but I would like to understand how this is achieved, is this because you used the more optimized tf-layers compared to tf-slim layers? Or do you not calculate everything that tf-slim inception calculates? I know that they also calculate/train aux-logits for example. In their code I also see that they have an additional average pooling part, is that also part of the hub module? 

So perhaps to clarify there are two parts of my question:
1. What is actually considered as the embedding in this module?
2. What causes the significant difference in performance, are these just optimizations on implementation or  should I expect some accuracy impact as well?.

Thanks for your help in understanding these results

**Comment by jdvylder** [comment]
OK after some digging found a bug in my code, apparently the tf-slim implementation was training more parameters than the module, explaining the bulk of the performance difference. Sorry for the confusion.
